agent/config/builder.go

```diff
@@ -1969,15 +1969,30 @@ func (b *builder) cidrsVal(name string, v []string) (nets []*net.IPNet) {
 	return
 }
 
-func (b *builder) tlsCipherSuites(name string, v *string) []uint16 {
+func (b *builder) tlsVersion(name string, v *string) types.TLSVersion {
+	// TODO: should the nil check be moved into ParseTLSVersion and the arguments
+	// switched to take a pointer?
```

Comment:
I think swapping `nil` for something more meaningful at the builder level is the right thing to do (no changes nessasary IMO) 👍🏻